### PR TITLE
Chinese Traditional Translation

### PR DESCRIPTION
**Pull request by open3**
**Details of Changes**
Add details of changes here.
- Updated the translation file

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

File: Files/MultilingualResources/Files.zh-Hant.xlf [hunk 1]
```diff
@@ -1,3466 +1,3446 @@
...
+        </trans-unit>
+        <trans-unit id="DrivesWidgetDescription.Text" translate="yes" xml:space="preserve">
+          <source>Drives</source>
+          <target state="final">磁碟機</target>
+        </trans-unit>
+        <trans-unit id="PinToSidebarByDraggingCaptionText" translate="yes" xml:space="preserve">
+          <source>Move here</source>
+          <target state="final">移至此處</target>
+        </trans-unit>
+        <trans-unit id="EjectNotificationHeader" tra
```
Comment (SiderealArt) [inline Files/MultilingualResources/Files.zh-Hant.xlf:3012]: Should be "特殊存取權限"

Comment (open3) [reply Files/MultilingualResources/Files.zh-Hant.xlf]: Fixed, Thanks!

File: Files/MultilingualResources/Files.zh-Hant.xlf [hunk 2]
```diff
@@ -1,3466 +1,3446 @@
...
+        </trans-unit>
+        <trans-unit id="DrivesWidgetDescription.Text" translate="yes" xml:space="preserve">
+          <source>Drives</source>
+          <target state="final">磁碟機</target>
+        </trans-unit>
+        <trans-unit id="PinToSidebarByDraggingCaptionText" translate="yes" xml:space="preserve">
+          <source>Move here</source>
+          <target state="final">移至此處</target>
+        </trans-unit>
+        <trans
```
Comment (SiderealArt) [inline Files/MultilingualResources/Files.zh-Hant.xlf:2111]:
```suggestion
          <target state="final">釘選至開始功能表</target>

```

Comment (SiderealArt) [inline Files/MultilingualResources/Files.zh-Hant.xlf:1619]:
```suggestion
          <target state="final">新增空檔案</target>

```

File: Files/MultilingualResources/Files.zh-Hant.xlf [hunk 3]
```diff
@@ -1,3466 +1,3446 @@
...
+          <source>Dark</source>
+          <target state="final">深色</target>
+        </trans-unit>
+        <trans-unit id="ApplicationTimeStye" translate="yes" xml:space="preserve">
+          <source>Application</source>
+          <target state="final">詳寫</target>
+        </trans-unit>
+        <trans-unit id="SystemTimeStye" translate="yes" xml:space="preserve">
+          <source>System</source>
+          <target state="final">簡寫</target>
```
Comment (SiderealArt) [inline Files/MultilingualResources/Files.zh-Hant.xlf:331]:
```suggestion
          <target state="final">系統</target>

```

File: Files/MultilingualResources/Files.zh-Hant.xlf [hunk 4]
```diff
@@ -1,3466 +1,3446 @@
...
+          <source>Light</source>
+          <target state="final">淺色</target>
+        </trans-unit>
+        <trans-unit id="DarkTheme" translate="yes" xml:space="preserve">
+          <source>Dark</source>
+          <target state="final">深色</target>
+        </trans-unit>
+        <trans-unit id="ApplicationTimeStye" translate="yes" xml:space="preserve">
+          <source>Application</source>
+          <target state="final">詳寫</target>
```
Comment (SiderealArt) [inline Files/MultilingualResources/Files.zh-Hant.xlf:327]:
```suggestion
          <target state="final">應用程式</target>

```

**Comment by open3** [comment]
All typos have been corrected.

File: Files/MultilingualResources/Files.zh-Hant.xlf [hunk 5]
```diff
@@ -1,3466 +1,3450 @@
...
+        </trans-unit>
+        <trans-unit id="DrivesWidgetDescription.Text" translate="yes" xml:space="preserve">
+          <source>Drives</source>
+          <target state="final">磁碟機</target>
+        </trans-unit>
+        <trans-unit id="PinToSidebarByDraggingCaptionText" translate="yes" xml:space="preserve">
+          <source>Move here</source>
+          <target state="final">移至此處</target>
+        </trans-unit>
+
```
Comment (SiderealArt) [inline Files/MultilingualResources/Files.zh-Hant.xlf:2591]:
```suggestion
          <target state="final">已取消刪除</target>

```

Comment (SiderealArt) [inline Files/MultilingualResources/Files.zh-Hant.xlf:2599]:
```suggestion
          <target state="final">已取消複製</target>

```

File: Files/MultilingualResources/Files.zh-Hant.xlf [hunk 6]
```diff
@@ -1,3466 +1,3452 @@
...
```
Comment (SiderealArt) [inline Files/MultilingualResources/Files.zh-Hant.xlf:2331]: Didn't translate "and {1} outgoing item(s).". Same for line 2335.

Comment (SiderealArt) [inline Files/MultilingualResources/Files.zh-Hant.xlf:2319]:
```suggestion
          <target state="final">顯示最近使用檔案小工具</target>

```

File: Files/MultilingualResources/Files.zh-Hant.xlf [hunk 7]
```diff
@@ -1,3466 +1,3452 @@
...
```
Comment (SiderealArt) [inline Files/MultilingualResources/Files.zh-Hant.xlf:2119]:
```suggestion
          <target state="final">從開始功能表取消釘選</target>

```

Comment (SiderealArt) [inline Files/MultilingualResources/Files.zh-Hant.xlf:3364]:
```suggestion
          <target state="final">已準備好安裝更新</target>

```

Comment (SiderealArt) [inline Files/MultilingualResources/Files.zh-Hant.xlf:3308]:
```suggestion
          <target state="final">搜尋檔案或資料夾時一併顯示未被索引的項目（可能會延長搜尋時間）</target>

```

Comment (SiderealArt) [inline Files/MultilingualResources/Files.zh-Hant.xlf:3295]:
```suggestion
          <target state="final">隱藏受保護的作業系統檔案（建議開啟）</target>

```

Comment (SiderealArt) [inline Files/MultilingualResources/Files.zh-Hant.xlf:3176]: May I ask why you changed the translation?

Comment (open3) [reply Files/MultilingualResources/Files.zh-Hant.xlf:3176]: Some strings in the v1.9.14 are not translate.

Comment (SiderealArt) [reply Files/MultilingualResources/Files.zh-Hant.xlf:3176]: sorry i didn't make myself clear. I'll ask again in Chinese.

可以問一下為甚麼你會想要修改 3176 行的翻譯嗎？

Comment (open3) [reply Files/MultilingualResources/Files.zh-Hant.xlf:3176]: 3176 行不是本來就沒有翻譯嗎 🤔
（您是指 "Select a destination and extract files" 嗎）

Comment (SiderealArt) [reply Files/MultilingualResources/Files.zh-Hant.xlf:3176]: 喔喔 不好意思我眼殘 等等幫您 Approve

**Review by SiderealArt — DISMISSED** [review]

File: Files/MultilingualResources/Files.zh-Hant.xlf (hunk 7 above)
Comment (open3) [reply Files/MultilingualResources/Files.zh-Hant.xlf:3176]: Thanks!

**Pull request closed without merging**